### PR TITLE
e2e testing release

### PR DESCRIPTION
**Pull request by ivarpruijn**
TODO:
- [x] Finalize docs & links in the blog post to those.

### CODE REVIEW

File: blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+          wait-on: 'http://localhost:3000'
+```
+
+
+This Action will run on every push, start the app and run the tests.
+
+# Not only regression testing
+The Tracking SDK now also exposes any tracking errors. One way to leverage this, is to write tests that load 
+every screen in your app and check those errors, so you can make sure your instrumentation fully passes 
+[validation][validation] everywhere.
```
Comment (sdirosa) [inline blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md:227]: Error testing is part of the snapshot testing described above. So, actually, by making sure e2e tests cover all events, one automatically tests also for errors for all of those cases.

File: blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+However, luckily you have your test. If you run the test locally while developing, or in your CI environment, 
+it will fail:
+
+<BlogImage url="/img/blog/releases/20220628/e2e-testing-fail.png" size="medium" />
+
+So this way, you’re immediately aware of any tracking regressions, and 
+**can solve them before they even end up in your codebase**.
+
+# How does it work?
+You can use any CI framework, but in this example app we use [Cypress][cypress]. Here’s the config:
```
Comment (sdirosa) [inline blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md:78]: I would change `the` to `our`, as the config we provide is just a very basic example. Most likely developers using cypress will have a much more complex one.

```suggestion
You can use any CI framework, but in this example app we use [Cypress][cypress]. Here’s our config:
```

**Comment by sdirosa** [comment]
@ivarpruijn Pretty nice! Left only 2 small suggestions. I'm going back to writing some docs, so we may complete this.

File: blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md (hunk 1 above)
Comment (ivarpruijn) [reply blog/2022-06-28 Release end-to-end testing of your tracking instrumentation.md:227]: Yes correct, though this just highlights how the new functionality can also be used in other ways.

**Comment by sdirosa** [comment]
@ivarpruijn I pushed a new section with the documentation about EventRecorder and how to set it up with Cypress under Tracking. 

Would be nice if you can give it a quick scan. And feel free to change anything you deem improvable of course. 

I have also some updates to do in React and React Native to reflect the changes I pushed for supporting React 18. Can I do that in this same PR or do you prefer a new one?

**Pull request merged**